r/R/python.R

```diff
@@ -181,9 +257,15 @@ r_to_py.RecordBatchReader <- function(x, convert = FALSE) {
   pa <- reticulate::import("pyarrow", convert = FALSE)
   x$export_to_c(stream_ptr)
   # TODO: handle subclasses of RecordBatchReader?
-  out <- pa$lib$RecordBatchReader$`_import_from_c`(
-    stream_ptr
-  )
+
+  if (pyarrow_version() >= pyarrow_version_pointers_changed()) {
```

Comment:
Will this actually compare version strings, such that "10.0.0" is smaller than "7.0.0"?